ui/common/css/abstract/_variables.scss

```diff
@@ -28,3 +28,17 @@ $site-header-height: var(--site-header-height);
 $site-header-margin: var(--site-header-margin);
 
 $site-header-outer-height: calc(#{$site-header-height} + #{$site-header-margin});
+
+$xxx-small: 400px;
+$xx-small: 500px;
+$x-small: 650px;
+$small: 800px;
+$medium: 980px;
+$large: 1120px;
+$x-large: 1260px;
+$xx-large: 1360px;
```

Comment:
I'm glad I checked! This value changed from 1500px to 1360px. Why? Are there any other functional changes in this PR?

Reply:
I think it was 1360px when this branch's code was actually forked and maybe became 1500px at some point recently?

Reply:
There are no intentional changes to any function, that was due to the block in question having a heritage in topnav-tweaks/_variables.scss which didn't pick up the corresponding mq-xx-large change in _media-queries.scss